runtime/drive_handler.go

```diff
@@ -125,10 +125,20 @@ func (h *stubDriveHandler) createStubDrive(driveID, path string) error {
 	return nil
 }
 
+func filterOnlyStubDrives(drives []models.Drive) []models.Drive {
```

Comment:
Let's move this function out of drive handler and into the `service.go`. You can then call the method before [set drives](https://github.com/firecracker-microvm/firecracker-containerd/blob/master/runtime/service.go#L651).

Reply:
While the latest version doesn't have filterOnlyStubDrives() anymore (as @xibz suggested), all changes are in service.go now.